governance/program/src/instruction.rs

```diff
@@ -410,10 +410,11 @@ pub enum GovernanceInstruction {
     ///
     ///   0. `[writable]` Realm account
     ///   1. `[signer]` Current Realm authority    
+    ///   2. `[]` New realm authority. Must be one of the realm governances when set
     SetRealmAuthority {
         #[allow(dead_code)]
-        /// New realm authority or None to remove authority
-        new_realm_authority: Option<Pubkey>,
+        /// Indicates whether the realm authority should be removed and set to None
+        remove_authority: bool,
```

Comment:
I prefer the intention to be explicitly specified using the flag because the authority could be accidentally removed if the additional account was not provided.